pkg/apis/operator/v1alpha1/operandregistry_types.go

```diff
@@ -240,8 +246,25 @@ func (r *OperandRegistry) GetAllReconcileRequest() []reconcile.Request {
 	return rrs
 }
 
-func init() {
-	SchemeBuilder.Register(&OperandRegistry{}, &OperandRegistryList{})
+// SetReadyCondition creates a Condition to claim Ready
+func (r *OperandRegistry) SetReadyCondition(name string, rt ResourceType, cs corev1.ConditionStatus) {
+	c := newCondition(ConditionReady, cs, string(rt)+" is ready", string(rt)+" "+name+" is ready")
+	r.setCondition(*c)
+}
+
+// SetNoneCondition creates a Condition to claim Ready
```

Comment:
line 249 and 255 are same, is this the expected?

Reply:
These are 2 different condition setup method, seems the comment need update